### PR TITLE
Feature/354 add company user

### PR DESCRIPTION
**Pull request by BlancoSebastianEzequiel**

### CODE REVIEW

File: test/graphql/CompanyUser/Mutations/saveCompanyUser.test.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  });
+
+  const performQuery = (apolloClient: TestClient, variables: ISaveCompanyUser) =>
+    apolloClient.query({ query: SAVE_COMPANY_USER, variables });
+
+  const generateVariables = ({ email, password }: IUserGeneratorAttributes = {}) => ({
+    user: UserGenerator.data.companyUser({ email, password })
+  });
+
+  it("creates a company user for the company of the current company user", async () => {
```
Comment (dylanalvarez) [inline test/graphql/CompanyUser/Mutations/saveCompanyUser.test.ts]: adds a company user to current user's company?

File: test/graphql/CompanyUser/Mutations/saveCompanyUser.test.ts [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+      userUuid: expect.stringMatching(UUID_REGEX),
+      user: {
+        uuid: expect.stringMatching(UUID_REGEX),
+        ...omit(variables.user, "password"),
+        dni: null
+      }
+    });
+  });
+
+  it("returns an error if the companyUser with the given email already exists", async () => {
```
Comment (dylanalvarez) [inline test/graphql/CompanyUser/Mutations/saveCompanyUser.test.ts]: if a companyUser

File: test/graphql/CompanyUser/Mutations/saveCompanyUser.test.ts [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+    const { apolloClient, user } = await TestClientGenerator.company({
+      status: ApprovalStatus.approved
+    });
+    const variables = generateVariables({ email: user.email });
+    const { errors } = await performQuery(apolloClient, variables);
+    expect(errors).toEqualGraphQLErrorType("UserEmailAlreadyExistsError");
+  });
+
+  it("returns an error if password does not hace digits", async () => {
```
Comment (dylanalvarez) [inline test/graphql/CompanyUser/Mutations/saveCompanyUser.test.ts]: if the password

have

File: test/graphql/CompanyUser/Mutations/saveCompanyUser.test.ts [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+    const { apolloClient } = await TestClientGenerator.company({
+      status: ApprovalStatus.approved
+    });
+    const variables = generateVariables({ password: "PasswordWithNoDigits" });
+    const { errors } = await performQuery(apolloClient, variables);
+    expect(errors).toEqualGraphQLErrorType("PasswordWithoutDigitsError");
+  });
+
+  it("returns an error if password has less than ten digits", async () => {
```
Comment (dylanalvarez) [inline test/graphql/CompanyUser/Mutations/saveCompanyUser.test.ts]: tha password

Comment (dylanalvarez) [reply test/graphql/CompanyUser/Mutations/saveCompanyUser.test.ts]: (idem el resto de los tests)

**Review by dylanalvarez — APPROVED** [review]

**Review by llmanuel — APPROVED** [review]

**Pull request merged**